### ISSUE TITLE
Topics pagination controls not accessible to screenreaders

### ISSUE THREAD

**Issue by lfrancis** [usnationalarchives/nara-bia-photos#146]
https://nara-bia-photos.herokuapp.com/topics/buildings?tribalNations[0]=Confederated%20Salish%20&%20Kootenai%20Tribes%20of%20the%20Flathead%20Reservation,%20Montana&tribalNations[1]=Crow%20Creek%20Sioux%20Tribe&tribalNations[2]=Duckwater%20Shoshone%20Tribe%20of%20the%20Duckwater%20Reservation,%20Nevada

It's unclear to screenreader users what the pagination button is for as it's not described as a pagination control via labeling:

![image](https://user-images.githubusercontent.com/6942589/96006150-2f8b0080-0e0b-11eb-8a59-86285c281e79.png)

Also, after selecting a new page from the pop-up, focus remains on the button, forcing the screenreader/keyboard user to fidntheir way back to the top of the new results listing -- ideally we'd do this focus resetting for them.
